index_api-v2.md

```diff
@@ -0,0 +1,191 @@
+# Murmurations Index API
+
+> :link: BASE URL
+>
+> _Test Environment_  
+> https://test-index.murmurations.network2/{endpoint}
```

Comment:
`https://test-index.murmurations.network2/{endpoint}` should be `https://test-index.murmurations.network/v2/{endpoint}`

Reply:
Good catch. Thanks!